HISTORY.rst

```diff
@@ -3,6 +3,14 @@
 Release History
 ===============
 
+0.13.1
++++++++++++++++
```

Comment:
The entries here are primarily for user facing changes which go in the user facing release notes. Since this is an engineering system improvement, you can omit them here. The PR message content, and the historical archive of them can be used to backtrack through these changes.